### PR TITLE
Fix datamodel warnings and roll deprecations

### PR DESCRIPTION
**Pull request by Jagusti**
Fixes for errors and warnings I've come across while working on v10 compatibility for GM Toolkit.

### CODE REVIEW

File: modules/system/audio-wfrp4e.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default class WFRP_Audio {
         console.warn("wfrp4e | Sound file not found for context: %o", context)
         return
       }
-      WFRP_UTILITY.log(`wfrp4e | Playing Sound: ${sound.file}`)
+      game.wfrp4e.utility.log(`wfrp4e | Playing Sound: ${sound.file}`)
```
Comment (moo-man) [inline modules/system/audio-wfrp4e.js:8]: Not that it really makes a difference, but why this change?

**Comment by Jagusti** [comment]
I think this one was because I was getting a WFRP_UTILITY error, as if it
wasn't being imported. I tried adding the import at the head of the file
and it gave me grief for some reason, so I fell back to this to avoid
getting more sidetracked!

I think there's another reference to WFRP_UTILITY at the bottom of this
file, but I've not amended that.  I've tried to only touch those nights of
code that I got an error called for. Hence a lot of the test rolls are not
explicitly async'd like I did basic test.
